src/EnergyPlus/DaylightingManager.cc

```diff
@@ -4308,6 +4322,7 @@ void GetInputIlluminanceMap(EnergyPlusData &state, bool &ErrorsFound)
         }
     }
     ZoneMsgDone.deallocate();
+    if (ErrorsFound) return;
```

Comment:
This was too late, and could cause an array bounds error in the eio printing below.